packages/kotti-ui/source/kotti-button/types.ts

```diff
@@ -18,17 +18,21 @@ export namespace KottiButton {
 	}
 	export const sizeSchema = z.nativeEnum(Size)
 
-	export const propsSchema = z.object({
-		icon: yocoIconSchema.nullable().default(null),
+	export const sharedSchema = z.object({
 		isBlock: z.boolean().default(false),
 		isLoading: z.boolean().default(false),
 		isMultiline: z.boolean().default(false),
 		isSubmit: z.boolean().default(false),
-		label: z.string().nullable().default(null),
 		size: sizeSchema.default(Size.MEDIUM),
 		type: typeSchema.default(Type.DEFAULT),
 	})
 
+	export const propsSchema = sharedSchema.extend({
```

Comment:
you don't need the split anymore

it can all go in sharedSchema now

it was like this because we were going to have two schemas extend separately from shared but since it's now one schema, please merge it back